src/experiments/buttonDesign/html.js

```diff
@@ -0,0 +1,11 @@
+/* @flow */
+
+export const html = ({ personalization } : {| personalization : {| text : string, tracking : {| impression : string, click : string |} |} |}) : string => {
+    if (personalization?.tracking?.impression) {
```

Comment:
I've added a Component for this now to prevent XSS, so don't need to add to `html` anymore.